mateog-assign1/crypto.py

```diff
@@ -0,0 +1,322 @@
+"""
+Assignment 1: Cryptography
+Course: CS 92SI
+Name: Mateo Garcia
+Date: 2015-10-11
+
+Encrypts and decrypts Caesar, Vigenere, and Railfence ciphers.
+"""
+
+import math
+
+alphabet = "ABCDEFGHIJKLMNOPQRSTUVWXYZ"
+
+def encrypt_caesar(plaintext):
+        """
+        Encrypts plaintext using a Caesar cipher. Maps alphabet to cipher
+        alphabet shifted by three characters to the left, such that A maps to D, etc.
+        Converts plaintext to list, and finds ciphertext version of each
+        character using map.
+        """
+        encrypt_alphabet = alphabet[3:] + alphabet[:3]
+        print(encrypt_alphabet)
+        encrypt_map = {}
+        for i in range (0, 26):
+                encrypt_map[alphabet[i]] = encrypt_alphabet[i]
```

Comment:
There's a more pythonic way to do this using the `zip` utility:
`encrypt_map = dict(zip(alphabet, encrypt_alphabet))`